src/main/kotlin/org/cultofclang/bonehurtingjuice/MoveListener.kt

```diff
@@ -58,8 +58,15 @@ internal object MoveListener : Listener {
             val higherBlock = it.add(0.0, 4.0, 0.0).block
             inBlock.isFlowing && higherBlock.isFlowing
         }?.let {
+
+            //bypass armor damage reduction
+            player.damage(0.0001) // trigger damage sound effect
+            if (player.health > 0.25 * Bones.waterfallDamgeMultiplier)
+                player.health = player.health - (0.25 * Bones.waterfallDamgeMultiplier)
```

Comment:
```suggestion
                player.health = (player.health - (0.25 * Bones.waterfallDamgeMultiplier)).coerceAtLeast(0.0)
```
No need for the if/else statement around it at all!

Reply:
oh pog you're right! It does work like that